package.json

```diff
@@ -24,12 +26,16 @@
     "node-sass": "^4.14.1",
     "pg": "^8.3.3",
     "pgtools": "^0.3.0",
-    "react": "^16.13.1",
+    "react": "^16.14.0",
     "react-bootstrap": "^1.3.0",
     "react-cookie": "^4.0.3",
-    "react-dom": "^16.13.1",
+    "react-device-detect": "^1.17.0",
+    "react-dom": "^16.14.0",
     "react-mapbox-gl": "^4.8.6",
+    "reactstrap": "^8.9.0",
     "request-ip": "^2.1.3",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^2.0.3",
```

Comment:
@lacabra I have removed the above unused packages in commit 7a9010f. I tried a couple different ways to display the information because of which I imported these packages, but finally I went with the modal. I forgot to remove the unused ones, thanks for pointing it out.